src/vivarium_csu_alzheimers/components/testing.py

```diff
@@ -36,51 +48,39 @@ def _create_testing_model(self) -> DiseaseModel:
             TESTING_ALZHEIMERS_DISEASE_MODEL.SUSCEPTIBLE_TO_TESTING,
             allow_self_transition=True,
         )
-        # TODO: can't get TransientState to work in a DiseaseModel
-        # testing = TransientState(
-        #     TESTING_ALZHEIMERS_DISEASE_MODEL.TESTING_STATE,
-        # )
+        testing = TestingState(TESTING_ALZHEIMERS_DISEASE_MODEL.TESTING_STATE)
         positive = DiseaseState(
             TESTING_ALZHEIMERS_DISEASE_MODEL.POSITIVE_STATE,
             prevalence=0.0,
             disability_weight=0.0,
             excess_mortality_rate=0.0,
         )
+        duration = pd.Timedelta(days=(365 * 2))  # 2 years
         negative = DiseaseState(
             TESTING_ALZHEIMERS_DISEASE_MODEL.NEGATIVE_STATE,
             prevalence=0.0,
             disability_weight=0.0,
             excess_mortality_rate=0.0,
+            get_data_functions={"dwell_time": lambda _, __: duration},
         )
 
         # Add transitions between states
         susceptible.add_transition(
+            output_state=testing,
+            probability_function=lambda index: pd.Series(TESTING_RATE, index=index),
+        )
+        testing.add_transition(
             output_state=positive,
-            probability_function=lambda index: pd.Series(
-                POSITIVE_TEST_TRANSITION_RATE, index=index
-            ),
+            probability_function=lambda index: pd.Series(POSITIVE_TEST_RATE, index=index),
         )
-        susceptible.add_transition(
+        testing.add_transition(
             output_state=negative,
-            probability_function=lambda index: pd.Series(
-                NEGATIVE_TEST_TRANSITION_RATE, index=index
-            ),
-        )
-        # testing.add_transition(
-        #     output_state=positive,
-        #     probability_function=lambda index: pd.Series(POSTIVE_TEST_RATE, index=index),
-        # )
-        # testing.add_transition(
-        #     output_state=negative,
-        #     probability_function=lambda index: pd.Series(1 - TESTING_RATE, index=index),
-        # )
-        negative.add_transition(
-            output_state=susceptible,
-            probability_function=lambda index: pd.Series(1.0, index=index),
+            probability_function=lambda index: pd.Series(1 - POSITIVE_TEST_RATE, index=index),
         )
+        negative.add_dwell_time_transition(output=susceptible)
```

Comment:
This is saying that people who test negative won't be tested again for 2 years, correct? At which point they become susceptible to getting tested again?